### PR TITLE
VGG Classification Model Implementation

### PR DESCRIPTION
**Pull request by RocketGrowthSJ**
1. VGG 모델로만 구현
2. training 과정과 예시는 script에 저장

기본적으로 Keras+Tensorflow(사실 Keras로만 일루어지긴했음)으로 구성했습니다.
이런 식으로 Localization 코드도 작성하고자 합니다.

### CODE REVIEW

**Comment by SpellOnYou** [comment]
I like your fast and clean code.

But, I want you to explain in more detail.

Here is my question

constructing the model, even though it is basic VGG network, I want you to comment about the parameter.

also, what is the 'tiny' VGG? is there 'enormous' model?

What are the criteria for being tiny? Is that mean it is a small number of pre-trained data parameters?

Why did you choose Adam as an optimizer function? Is not there another option? likewise, loss function.

II do not want you to explain each one in detail, but I would like you to write a reason for the choice.

I will merge but wanna more explain.

@RocketGrowthSJ
